src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -50,7 +50,7 @@ DLT_STATIC void dlt_logstorage_concat_logfile_name(char *log_file_name, const ch
     size_t src_len = strlen(append);
 
     if (dst_len < DLT_MOUNT_PATH_MAX) {
-        size_t rem_len = DLT_MOUNT_PATH_MAX - dst_len - 1;
+        size_t rem_len = DLT_MOUNT_PATH_MAX - dst_len;
```

Comment:
Please revert this change. 
strncat() below expects:  "destination buffer large enough, that is, strlen(dst) + strnlen(src, sz) + 1." [1] 


[1] https://man7.org/linux/man-pages/man3/strncat.3.html